### ISSUE TITLE
PyRevitLoader - Error Loading pyRevit 

### ISSUE THREAD

**Issue by Stacey-84** [pyrevitlabs/pyRevit#2305]
### ✈ Pre-Flight checks

- [X] I **don't have SentinelOne** antivirus installed (see above for the solution)
- [X] I have **searched in the issues** (open and closed) but couldn't find a similar issue
- [X] I have **searched in the [pyRevit Forum](https://discourse.pyrevitlabs.io)** for similar issues
- [X] #2306
- [X] I am using the **latest pyRevit Version**

### 🐞 Describe the bug

pyRevit Loader runtime error, fails to launch pyRevit. 

Unistalled and reinstalled, unistalled revit and reinstalled, no solution, same error. 

The issue was when I tried installing pyRevit on Revit 2024, was always working fine on previous Revit versions. 

Please help. Thanks in advance. 

### ⌨ Error/Debug Message

```shell
Searched numerous workarounds but nothing worked.
```

### 🖥️ Hardware and Software Setup (please complete the following information)

```shell
Microsoft Windows [Version 10.0.22631.3810]
(c) Microsoft Corporation. All rights reserved.

C:\Users\Stacey Ferguson> pyrevit env
==> Registered Clones (full git repos)
==> Registered Clones (deployed from archive/image)
master | Deploy: "basepublic" | Branch: "master" | Version: "4.8.16.24121+2117" | Path: "C:\Users\Stacey Ferguson\AppData\Roaming\pyRevit-Master"
==> Attachments
master | Product: "24.2.1" | Engine: DEFAULT (2711) | Path: "C:\Users\Stacey Ferguson\AppData\Roaming\pyRevit-Master"
master | Product: "23.1.4" | Engine: DEFAULT (2711) | Path: "C:\Users\Stacey Ferguson\AppData\Roaming\pyRevit-Master"
master | Product: "2022 First Customer Ship" | Engine: DEFAULT (2711) | Path: "C:\Users\Stacey Ferguson\AppData\Roaming\pyRevit-Master"
==> Installed Extensions
==> Default Extension Search Path
C:\Users\Stacey Ferguson\AppData\Roaming\pyRevit\Extensions
==> Extension Search Paths
==> Extension Sources - Default
https://github.com/pyrevitlabs/pyRevit/raw/master/extensions/extensions.json
==> Extension Sources - Additional
==> Installed Revits
24.2.1 | Version: 24.2.10.64 | Build: 20240408_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2024\"
23.1.4 | Version: 23.1.40.56 | Build: 20240411_0945(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2023\"
2022 First Customer Ship | Version: 22.0.2.392 | Build: 20210224_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2022\"
==> Running Revit Instances
PID: 23320 | 2022 First Customer Ship | Version: 22.0.2.392 | Build: 20210224_1515(x64) | Language: 0 | Path: "C:\Program Files\Autodesk\Revit 2022"
==> User Environment
Microsoft Windows 10 [Version 10.0.22631]
Executing User: A3ARCHITECTS\Stacey Ferguson
Active User: A3ARCHITECTS\Stacey Ferguson
Admin Access: No
%APPDATA%: "C:\Users\Stacey Ferguson\AppData\Roaming"
Latest Installed .Net Framework: 4.8
No .Net Target Packs are installed.
No .Ne-Core Target Packs are installed.
pyRevit CLI v4.8.16.24121+2117.23a0b8cbc4babdeb29a0611e159d017bbe9cbad5

C:\Users\Stacey Ferguson>
```

**Comment by sanzoghenzo** [comment]
> I already **followed the [installation troubleshooting guide](https://pyrevitlabs.notion.site/Installation-issues-f6495adb56254e9e8021f8a68e3b2ab7) thoroughly**

Have you _really_ followed it?

**Comment by SpaceArchitecture** [comment]
Same problem with me, it seems the installer for the latest version doesn't like Revit 2024....
Too bad

**Comment by sanzoghenzo** [comment]
This is a known issue, and possible solutions are shown in the [installation troubleshooting guide](https://pyrevitlabs.notion.site/Installation-issues-f6495adb56254e9e8021f8a68e3b2ab7).

Until we have a confirmation that that guide has in fact being followed, and more info about the error is reported, we can't guess and provide any more info.

**Comment by SpaceArchitecture** [comment]
It's a known issue ? Is somebody working on a fix ? It seems pretty widespread.

**Comment by sanzoghenzo** [comment]
TL:DR: Yes; in our free, unpaid time; yes we know.

Long version:
- as I wrote earlier Yes, it is a known problem and many of the closed issues are a variation on this theme.
- the resolution of this problem is blocked by the ongoing changes related to supporting the new revit versions, but we have little to no time/resources to finish that (@dosymep did almost all the hard work, I don't have access to revit 2025, nobody else is trying to help)
- we know that it is widespread since we have to reply to each of the issues with the same answer 😉 the troubleshooting guide provides a workaround, but it seems that nobody reads it. The issues should be used to report problems only if the provided workarounds are not effective.

**Comment by aussieBIMguru** [comment]
If it helps, the WIP build I installed last week is working fine in Revit 2024, although I have implemented the revit application fixed we discussed. Not sure if that helps identify the potential changes that cause the issues in that specific build?
